src/model/core/co2.jl

```diff
@@ -0,0 +1,108 @@
+@doc raw""" 
+
+co2!(EP::Model, inputs::Dict)
+
+This function creates expression to account for CO2 emissions and captured and sequestrated CO2 from thermal generators. It also has the capability to model the negative CO2 emissions from bioenergy with carbon capture and storage. This module will displace the emissions module.
```

Comment:
Please remove the last sentence which only refers to a change in the code.

Reply:
ok! And I've added new cols to the MD files